droidlet/dashboard/web/src/components/MobileAnnotationComponents/ImageDrawer.jsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import DataEntry from "./DataEntry";
+
+class ImageDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.canvasRef = React.createRef();
+    this.imgRef = React.createRef();
+    this.isDrawing = false;
+    this.annotationName = null;
+    this.annotationTags = null;
```

Comment:
why are some of these null and others empty list?

Reply:
Name is a string, tag is an array. will change